Cargo.toml

```diff
@@ -51,6 +52,7 @@ thiserror = "1.0.16"
 serde = "1.0.106"
 bytes = "1.5.0"
 tokio = { version = "1.0.1", features = ["rt", "macros"] }
+hyper-rustls = { version = "0.24", features = ["http2"], optional = true }
```

Comment:
As far as I know, CH doesn't support http2, so it's useless to enable it anyway.

Reply:
Gotcha -- I misread the hyper-rustls docs and thought that it was required to use `wrap_connector`, but it's not, so I moved back to plain HTTP1.